graphql/schema.graphql

```diff
@@ -363,7 +363,7 @@ input ProjectSettingsInput {
 }
 
 input ProjectInput {
-  id: String!
+  id: String
```

Comment:
Why the change to make this not required?

Reply:
This fails for CreateProject, because it also uses ProjectInput and ID wouldn't be passed in in that case.

Reply:
Should I switch so create project uses like a CreateProjectInput or something?

Reply:
Ah that makes sense. duplicating it feels overkill, I wouldn't do that. maybe it's okay for it not to be required. @sophstad any thoughts on this?